pages/common/devenv.md

```diff
@@ -0,0 +1,28 @@
+# devenv
+
+> Fast, Declarative, Reproducible and Composable Developer Environments using Nix.
+> Website/Docs: <https://devenv.sh/>. GitHub repository: <https://github.com/cachix/devenv>.
+
+- Initialise environment:
+
+`devenv init`
+
+- Enter Development Environment with relaxed hermeticity:
```

Comment:
```suggestion
- Enter the Development Environment with relaxed hermeticity:
```

Reply:
```suggestion
- Enter the Development Environment with relaxed hermeticity (state of being airtight):
```
Hermeticity is a new word for me and a few others. Since it's jargon maybe we can add its meaning beside it